sdk/forecast-2019-07-22.normal.json

```diff
@@ -0,0 +1,1635 @@
+        "FeaturizationConfig"
+      ],
+      "members":{
+        "PredictorName":{
+          "shape":"Name",
+          "documentation":"<p>A name for the new predictor. </p>"
+        },
+        "AlgorithmArn":{"shape":"Arn"},
+        "ForecastHorizon":{
+          "shape":"Integer",
+          "documentation":"<p>Specifies the number of forecasts to make. The forecast horizon is also called the prediction length.</p> <p>For example, if you configure a dataset for daily data collection (using the <code>DataFrequency</code> parameter of the <a>CreateDataset</a> operation) and set the forecast horizon to 10, the model returns forecasts for 10 days.</p>"
+        },
+        "PerformAutoML":{
+          "shape":"Boolean",
+          "documentation":"<p>The default value is <code>N</code>. You are required to choose a recipe.</p> <p>If you want Amazon Forecast to evaluate the recipes it provides (not custom recipes) and choose the best recipe and configuration for your training dataset, set <code>PerformAutoML</code> to <code>Y</code>. This is a good option if you aren't sure which recipe is suitable for your application.</p>"
+        },
+        "PerformHPO":{
+          "shape":"Boolean",
+          "documentation":"<p>Whether to perform hyperparameter optimization. The default value is <code>Y</code>.</p> <p>A hyperparameter tuning job finds optimal hyperparameter values for your training data. If you specify <code>HPOConfig</code>, the tuning job works on the ranges that you provide. Otherwise, it works with the default hyperparameter ranges.</p> <p>The following recipes support HPO:</p> <ul> <li> <p>DeepAR+</p> </li> </ul>"
+        },
+        "TrainingParameters":{
+          "shape":"TrainingParameters",
+          "documentation":"<p>The training parameters to override for model training. The parameters that you can override are listed in the individual recipes in <a>aws-forecast-choosing-recipes</a>.</p>"
+        },
+        "EvaluationParameters":{"shape":"EvaluationParameters"},
+        "HPOConfig":{
+          "shape":"HyperParameterTuningJobConfig",
+          "documentation":"<p>Provides hyperparameter override values for the algorithm in the specified recipe. If you don't provide this parameter, Amazon Forecast uses default values.</p>"
+        },
+        "InputDataConfig":{"shape":"InputDataConfig"},
+        "FeaturizationConfig":{"shape":"FeaturizationConfig"},
+        "EncryptionConfig":{"shape":"EncryptionConfig"}
+      }
+    },
+    "CreatePredictorResponse":{
+      "type":"structure",
+      "members":{
+        "PredictorArn":{
+          "shape":"Arn",
+          "documentation":"<p>The Amazon Resource Name (ARN) of the predictor.</p>"
+        }
+      }
+    },
+    "DataDestination":{
+      "type":"structure",
+      "members":{
+        "S3Config":{"shape":"S3Config"}
+      }
+    },
+    "DataSource":{
+      "type":"structure",
+      "members":{
+        "S3Config":{"shape":"S3Config"}
```

Comment:
Is this the latest JSON? I know Venkat marked this field as required in the latest JSON? This happened today.

Reply:
Can we also remove the old JSON files from the Repo if not being referenced?

Reply:
Yeah he pinged me. He committed sometime in the afternoon. I will pick it up. Shouldn't change the tests though. Will also remove the old json files